server/api/impl/aws/provision_app.go

```diff
@@ -46,20 +53,21 @@ type functionInstallData struct {
 //      |-- __pycache__
 //      |-- certifi/
 func (c *Client) ProvisionApp(releaseURL string) error {
-	zipFile, err := downloadFile(releaseURL)
-	if err != nil {
-		return errors.Wrapf(err, "can't install app from url %s", releaseURL)
+	zipFile, zipErr := downloadFile(releaseURL)
```

Comment:
2/5 on using the word `bundle` for the zip file
```suggestion
	bundle, bundleErr := downloadFile(releaseURL)
```